inc/class-contributors.php

```diff
@@ -406,10 +420,13 @@ public function personalName( $slug ) {
 		$name = '';
 		$term = get_term_by( 'slug', $slug, self::TAXONOMY );
 		if ( $term ) {
+			$prefix = get_term_meta( $term->term_id, 'contributor_prefix', true );
+			$suffix = get_term_meta( $term->term_id, 'contributor_suffix', true );
 			$first_name = get_term_meta( $term->term_id, 'contributor_first_name', true );
 			$last_name = get_term_meta( $term->term_id, 'contributor_last_name', true );
 			if ( ! empty( $first_name ) && ! empty( $last_name ) ) {
-				$name = "{$first_name} {$last_name}";
+				$name = $prefix ? "{$prefix} {$first_name} {$last_name}" : "{$first_name} {$last_name}";
+				$name = $suffix ? "${name} $suffix" : $name;
```

Comment:
The convention is to add a comma between name and suffix, unless it's a roman numeral. For example, we should write `Ken Griffey, Jr.` and `Ted Logan, Esq.` but `Alec Baldwin III`. Can we create rule to handle this that looks for `III` or `IV` or strings that start with `V` and contain 1-3 `I`s? Should cover most use cases.